core/src/components/datetime/datetime.tsx

```diff
@@ -313,6 +314,14 @@ export class Datetime implements ComponentInterface {
    */
   @Prop() firstDayOfWeek = 0;
 
+  /**
+   * A callback used to format the header text that shows how many
+   * dates are selected. Only used if there are 0 or more than 1
+   * selected (i.e. unused for exactly 1). By default, the header
+   * text is set to "datesSelected days".
+   */
+  @Prop() selectedDatesLabelFormatter?: SelectedDatesLabelFormatter;
```

Comment:
I'm not married to this name if anyone wants to suggest alternatives.

Reply:
Perhaps `headerTextFormatter` or `headerSelectedDatesFormatter`?

Reply:
Hmm, maybe `titleSelectedDatesFormatter` to match up with the `showDefaultTitle` prop?

Reply:
That works for me 👍 I like aligning context with `showDefaultTitle`.